pure_pursuit:
  start_time: 5
  rad_tolerance: 0.1 # radius fidelity of the smoothing
  tolerance: 0.115 # [m] tolerance
  alpha: 0.60 # curve constant
  beta: 0.40 # curve constant
  w_base: 0.23  # [m] wheel base
  spacing: 0.025 # [m] space between injected points
  kv: 1.0 # velocity constant
  ka: 0.0125 # acceleration constant
  lad: 0.275  # [m] look-ahead distance
  lfg: 0.0625 # look forward gain
  target_speed: 0.15 # [m/s] velocity target
  filter_size: 5 # size of the averaging filter
  kp: 2.5 # pid proportional gain
  ki: 0.0 # integral gain
  kd: 0.0 # derivative gain
  error_bound: 0.0 # a hard cap on the pid error. if 0.0, no cap